nservicebus/dotnet-templates/index_nsbendpoint_templates_[5,).partial.md

```diff
@@ -0,0 +1,41 @@
+## NServiceBus Endpoint
+
+The `nsbendpoint` template will create a C# project for an NServiceBus endpoint. By selecting options, the endpoint is preconfigured with choices of:
+
+*  Target framework
+*  [Message transport](/transports/)
+*  [Data persistence](/persistence/)
+*  [Hosting model](/nservicebus/hosting/), which can be one of:
+   * Console Application
+   * [Windows service hosting](/nservicebus/hosting/#self-hosting-windows-service-hosting)
+   * [Docker container hosing](/nservicebus/hosting/docker-host)
+
+Once the package is installed, the template can be found in the Visual Studio **New Project** dialog:
+
+![NServiceBus endpoint template in the Visual Studio New Project dialog](new-project.png)
+
+After naming the project and selecting a location for it on disk, the **Additional information** screen will allow customizing the template:
+
+![Customing endpoint properties in the additional information dialog](endpoint-additional-info.png)
+
+### Command-line usage
+
+The default usage will create a Console Application using Learning Transport, Learning Persistence:
+
+snippet: endpointdefault
+
+Parameters allow the selection of message transport, data persistence, and hosting model, including [Windows Service](/nservicebus/hosting/#self-hosting-windows-service-hosting) or [Docker container](/nservicebus/hosting/docker-host/) hosting.
+
+### Options
+
+| Option | Description |
```

Comment:
Add serializer?

Reply:
It's not an option. It just does System.Text.Json

Reply:
ah, got it now 👍